content/en/news/releases/1.15.x/announcing-1.15.5/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Announcing Istio 1.15.5
+linktitle: 1.15.5
+subtitle: Patch Release
+description: Istio 1.15.5 patch release.
+publishdate: 2023-1-30
```

Comment:
I think we need to make this 2023-01-30 as when looking at the deploy output, I do not see the news item.